build_tools/python/benchmark_suites/iree/benchmark_tags.py

```diff
@@ -0,0 +1,18 @@
+## Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Tag that indicates this compile config is generated for collecting compilation
+# statistics.
+COMPILE_STATS = "compile-stats"
+
+# Tag for long-running benchmarks.
+LONG_RUNNING = "long-running"
```

Comment:
Especially when speaking about compilation of the models, is "long running" really the right name? That to me implies execution time. It seems like "large" would be simpler, shorter, and more accurate

Reply:
Talked to @mariecwhite, I think we can rename this to `large` . I can do in a follow-up change.